compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BinaryFileWriter.java

```diff
@@ -222,7 +222,7 @@ private void createDirectory(Path tempDir) {
         try {
             Files.createDirectories(tempDir);
         } catch (IOException e) {
-            throw new BLangCompilerException("failed create directory '" + tempDir.toString() + "'", e);
+            throw new BLangCompilerException("failed to create directory '" + tempDir.toString() + "'", e);
```

Comment:
Do you think it would be nice to have first letter as capital:
```suggestion
            throw new BLangCompilerException("Failed to create directory '" + tempDir.toString() + "'", e);
```

Reply:
We usually start diagnostic messages with lower-case letters. I think we should continue to do the same here. We can change the error in `BIRPackageSymbolEnter.java` too.